types/pretty-ms/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for pretty-ms 3.2
+// Type definitions for pretty-ms 4.0
```

Comment:
Please follow the instructions in the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version) to introduce a major version update.

Reply:
Fixed.

Reply:
Thank you!